docs/web/index.md

```diff
@@ -534,14 +532,9 @@ management.endpoint.health.show-components=always
 
 ## Logging
 
-The Spring Boot default is enabled. 
-
-### Configuration properties
-
-* `logging.config` _string_
-  * Path to the logback configuration on classpath. If not set, default console Spring logging is
-    active.
-    Available custom configurations on classpath:
-    * `classpath:org/sdase/commons/spring/logging/logback-json.xml` for Json Logging
-  * Example: `classpath:org/sdase/commons/spring/logging/logback-json.xml`
-  * Default: `org/springframework/boot/logging/logback/defaults.xml`
+The Spring Boot default logging is enabled.
+Logs are printed to standard out.
+`ENABLE_JSON_LOGGING=true` as environment variable or `-Denable.json.logging=true` as JVM parameter
+enables output as JSON for structured logs used in log aggregation tools.
+To enable JSON logging in `application.(properties/yaml)`,
+`logging.config=classpath:org/sdase/commons/spring/boot/web/logging/logback-json.xml` may be used. 
```

Comment:
No, because we need the information before spring discovers property sources.
Imho we should not bloat bloat the docs with background information. Is the fact that `enable.json.logging` **won't work** in `application.properties` clear enough described?